docs/ARCHITECTURE.md

```diff
@@ -39,23 +39,23 @@ _Depends on: nothing_
 ### Data pipeline 📥➡️♨➡️📤
```

Comment:
IMHO this document could use a few simple sentences at the top of the document describing how many of the classes for GTFS loading and validation are auto-generated using AutoValue and annotations. This is really the huge difference between v1 and v2 and it's somewhat buried in the bullets right now.